app/views/layouts/application.html.erb

```diff
@@ -1,4 +1,12 @@
 <% content_for(:head) do %>
+  <% if @transient_registration.present? %>
+  <script>
+    dataLayer = [{
+      <%= data_layer(@transient_registration) %>
+    }];
+  </script>
```

Comment:
Can we indent this block?

Reply:
Resolved with https://github.com/DEFRA/waste-exemptions-front-office/pull/194/commits/3016f9b54aaa55f0b8b4c261b83450f684a8d81c